src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterConstants.java

```diff
@@ -0,0 +1,29 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+public class LifterConstants {
+
+    public static final Translation3d ROBOT_RELATIVE_LIFTER_POSITION = new Translation3d(0,0.24,0.35);
+
+
+    public static final Rotation3d ROBOT_RELATIVE_LIFTER_ROTATION = new Rotation3d(Math.PI,0,-Math.PI/2);
+
+    public static final Rotation2d ENCODER_POSE_WHEN_RESET = Rotation2d.fromRotations(0);
+
+    public static final double TOLERANCE = 0.04;
```

Comment:
remember that the tolerance is an angle. consider changing the tolerance to a rotation 2d